src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss

```diff
@@ -0,0 +1,139 @@
+@import "./../../../../assets/stylesheets/custom";
+@import "./../../../../assets/stylesheets/mixins";
+@import "./../../../../assets/stylesheets/color-variables";
+.detail-container {
+  width: 100%;
+  float: left;
+  .note{
+    color: #cccccc;
```

Comment:
@SMahil can you please look into these questions that @mindreeper2420 has raised.